x-pack/test_serverless/api_integration/test_suites/common/data_views/default_index_pattern/default_index_pattern.ts

```diff
@@ -69,7 +69,10 @@ export default function ({ getService }: FtrProviderContext) {
             .get(defaultPath)
             // TODO: API requests in Serverless require internal request headers
             .set(svlCommonApi.getInternalRequestHeader());
-          expect(response6.body[serviceKeyId]).to.be(null);
+          // TODO: The response comes back undefined in Serverless
+          const body = response6.body[serviceKeyId];
+          const expected = body === undefined ? null : body;
+          expect(expected).to.be(null);
```

Comment:
@mattkime do you think this is ok? Clearing the default data view works in Serverless by passing `null`, but the return value after fetching is now `undefined` instead of `null` as it is in stateful.